Gruntfile.coffee

```diff
@@ -1227,11 +1208,15 @@ module.exports = (grunt) ->
     'jasmine_chromeapp:socksEcho'
   ]
 
+  grunt.registerTask 'unit_test_nobuild', _.flatten(
```

Comment:
Great catch @fortuna! Jeez, that seems like such an easy mistake to make, but I guess it would be asking too much of lodash to warn on `arguments.length > 1` (and can't help but wish javascript helped with passing more args to a function than its signature has, at least with e.g. `"use strict";`, but not so much)